src/authentication.ts

```diff
@@ -33,4 +33,18 @@ export const expressAuthentication = mergeAcceptAny([
       return scopes.split(' ')
     },
   }),
+  mkExpressAuthentication({
+    verifyOptions: {},
+    securityName: 'external',
+    jwksUri: () =>
+      Promise.resolve(
+        `${env.IDP_INTERNAL_ORIGIN}${env.IDP_PATH_PREFIX}/realms/${env.IDP_EXTERNAL_REALM}/protocol/openid-connect/certs`
+      ),
+    getAccessToken: (req: express.Request) =>
+      Promise.resolve(req.headers['authorization']?.substring('bearer '.length)),
+    getScopesFromToken: async (decoded) => {
+      const scopes = typeof decoded === 'string' ? '' : `${decoded.scopes}`
```

Comment:
found out through my current ticket that it's `scope` on jwt rather than `scopes`